app/tests/api/scripts/test_create_eligibility_screener_csv.py

```diff
@@ -0,0 +1,161 @@
+import csv
+
+import pytest
+
+from api.db.models.factories import EligibilityScreenerFactory
+from api.scripts.create_eligibility_screener_csv import (
+    DATETIME_OUTPUT_FORMAT,
+    ELIGIBILITY_SCREENER_CSV_HEADER,
+    EligiblityScreenerCsvRecord,
+    convert_eligibility_screener_records_for_csv,
+    create_eligibility_screener_csv,
+    get_eligibility_screener_records,
+)
+from api.util.datetime_util import utcnow
+from api.util.string_utils import blank_for_null, join_list
+
+
+@pytest.fixture
+def tmp_file_path(tmp_path):
+    return tmp_path / "example_file.csv"
+
+
+def read_csv_records(file_path):
+    with open(file_path) as csv_file:
+        csv_reader = csv.DictReader(csv_file)
+        csv_rows = list(csv_reader)
+        return csv_rows
+
+
+def validate_csv_records(db_records, csv_records, test_db_session):
+    assert len(csv_records) == len(db_records)
+
+    # Sort the two lists by first name and zip together for validation
+    csv_records.sort(key=lambda record: record["First Name"])
+    db_records.sort(key=lambda record: record.first_name)
+    for csv_record, db_record in zip(csv_records, db_records):
+        test_db_session.refresh(db_record)
```

Comment:
Can you remind me what `refresh()` does again at this point?

Reply:
It updates the object with whatever has been flushed to the DB. In this case, the refresh is to get the timestamp that was added to another object representing the same DB record.